CONTRIBUTING.md

```diff
@@ -80,18 +80,53 @@ CI/CD will run tests, and if successful, do deploys.
 Documentation is built and deployed, and code snippets
 are exported to the service contract repo as a pull request.
 
-Updating and uploading npm packages only happens if the HEAD commit of the main branch
-contains `[release]` in its description or the PR title starts with `feat` or `fix`.
+Updating and uploading npm packages only happens if the HEAD commit of the main branch or a release-* branch
+contain `[release]` and `[bump-version]` in its description and the PR title starts with `feat` or `fix`.
+
+#### Examples
+
+1. If you want to release a release candidate version to npm with version bump when you are commiting to a release-* branch.
+```bash
+  feat(dataParser): add xml support [release][bump-version]  
+  fix(graphUtils): circular path detection [release][bump-version] 
+```
+2. If you want to release a release candidate version to npm without version bump when you are commiting to a release-* branch.
```

Comment:
Why would you want to do this? In what scenario do you want to release a change but not have at least a patch bump? and Would you ever do a feat release and not bump the version?

Reply:
If I understood correctly, this was the main concern that needed to be addressed according to the previous meeting. 
It is such that if we make small changes in the release branch (such as test case adjustment and merge then it will update the version number, which was unintentional (if my understanding is correct).

Your concern here is valid since someone mistakenly did merge a significant change with [release] tag which is intentionally need to adhere to the versioning convention then it won't reflect without [bump-verison] tag.

Reply:
but would you not use test( ) if you are updated testing? Maybe we should include this as examples? As updating tests are not patch as far as I know